packages/block-library/src/heading/style.scss

```diff
@@ -7,7 +7,8 @@
 	h6 {
 		background-color: var(--wp--color--background);
 		color: var(--wp--color--text);
-		line-height: var(--wp--typography--line-height);
+		line-height: var(--wp--typography--line-height, var(--wp--typography--line-height-heading));
+		font-weight: var(--wp--typography--font-weight-heading);
```

Comment:
Should we just remove the "-heading" suffix given that we will probably not have block specific variables?

Reply:
I've commented on this at https://github.com/WordPress/gutenberg/pull/20530#discussion_r400836040 TLDR is that so far we don't have a way to do otherwise. I proposed a [potential direction](https://href.li/?https://github.com/WordPress/gutenberg/pull/20290) but it needs some adjustments before is ready.

This is a discussion that also needs to address what do we do with font-size (used at the custom styles/block level) VS font base/scale (used at the global styles level) I mentioned at https://github.com/WordPress/gutenberg/pull/20530#discussion_r400840203

Those seem conversations we need to have before settling on anything, so I lean towards merging this and then iterate. How does it sound?